modules/ROOT/pages/release-notes/release-notes5013.adoc

```diff
@@ -24,14 +24,14 @@ TinyMCE now supports all valid HTML color formats for the `color_map` setting. F
 
 NOTE: The previous hex code color format without the "#" is still accepted as valid values.
 
-For more information on the `color_map` setting, refer to the link:{baseurl}/configure/content-appearance/#color_map[documentation].
+For more information on the `color_map` setting, refer to the xref:configure/content-appearance.adoc#color_map[documentation].
```

Comment:
This xref doesn't resolve as the `color_map` id doesn't exist on the relevant header in `text_color.adoc`. e.g:
```
[[color_map]]
=== `color_map`
```

Reply:
done